portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,10 +61,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Map<String, Map<String, String>> comments = new HashMap<>();
     for (Entity entity : results.asIterable()) {
       String id = KeyFactory.keyToString(entity.getKey());
-      Map<String, String> commentEmail = new HashMap<>();
-      commentEmail.put("comment", (String) entity.getProperty("comment"));
-      commentEmail.put("userEmail", (String) entity.getProperty("userEmail"));
-      comments.put(id, commentEmail);
+      Map<String, String> commentAttributes = new HashMap<>();
+      commentAttributes.put("comment", (String) entity.getProperty("comment"));
```

Comment:
As all attributes are related to a comment, should the this be 'text' or 'commentText'?

Reply:
Yes this should be commentText, thanks for picking up on this.